src/main/java/uk/gov/justice/digital/delius/service/CustodyService.java

```diff
@@ -59,4 +75,14 @@ public Custody updateCustody(final String nomsNumber,
             return new NotFoundException(String.format("offender with nomsNumber %s not found", nomsNumber));
         });
     }
+
+    private Event updateInstitutionOnEvent(Event event, RInstitution institution) {
+        if (updateCustodyFeatureSwitch) {
+            event.getDisposal().getCustody().setInstitution(institution);
```

Comment:
WIP - this is where we will do the countless calls to update the event, create contacts etc etc